tests/test_dates.py

```diff
@@ -16,27 +16,38 @@
 
 import freezegun
 import pytest
-import pytz
-from pytz import timezone
+
+# for tests it can be useful to have both zoneinfo and pytz available
+try:
+    import zoneinfo
+except ModuleNotFoundError:
+    try:
+        from backports import zoneinfo
+    except ImportError:
+        zoneinfo = None
+
+try:
+    import pytz
+except ModuleNotFoundError:
+    pytz = None
 
 from babel import dates, Locale
-from babel.dates import NO_INHERITANCE_MARKER
+from babel.dates import NO_INHERITANCE_MARKER, get_timezone, localize, _get_tz_name, LOCALTZ
```

Comment:
Hmm... I don't think we should use `get_timezone` in these tests at all, but use the `timezone_getter` fixture wherever we would?